### PR TITLE
Update application-menus.html.erb.md

### PR DESCRIPTION
**Pull request by nitedog**

### CODE REVIEW

File: source/menus/application-menus.html.erb.md [hunk 1]
```diff
@@ -26,298 +22,11 @@ In addition to the `aria-expanded` and `aria-haspopup` attributes, the following
 
 {::nomarkdown}<%= ref :middle %>{:/}
 
-To keep this example simple, sub-submenus are not shown in this tutorial.
+Application menus typically do not have links and rely on scripting to provide the functionality.
```
Comment (nitedog) [inline source/menus/application-menus.html.erb.md]: OK. But "sub-sub" reads kind of odd. Maybe "second-level menus" or such?

File: source/menus/application-menus.html.erb.md [hunk 2]
```diff
@@ -345,35 +54,13 @@ The markup has no links at all: It is a nested list with WAI-ARIA roles as the a
 
 ## Functionality
 
-The behavior of desktop application menus is added to the menu by this JavaScript. First, we collect all top-level menu items in one variable (`appsMenuItems`) and all submenu items in another (`subMenuItems`). Two variables keep track of the focus in top-level items (`currentIndex`) and in submenus (`subIndex`).
+Functionality and keyboard behavior are **not** automatically enabled when WAI-ARIA roles are added. These need to be added using scripting. A detailed explanation on the WAI-ARIA attributes and keyboard behavior can be found in the [WAI-ARIA Authoring Practices document (draft)](http://www.w3.org/TR/wai-aria-practices/#menu).
```
Comment (nitedog) [inline source/menus/application-menus.html.erb.md]: s/These need/Functionality needs

File: source/menus/application-menus.html.erb.md [hunk 3]
```diff
@@ -345,35 +54,13 @@ The markup has no links at all: It is a nested list with WAI-ARIA roles as the a
...
-	right:  39,
-	down:   40
-};
-var currentIndex, subIndex;
-~~~
-
-{::nomarkdown}
-<%= code_end %>
-{:/nomarkdown}
+Web application menus are expected to work like desktop application menus. For example, the tab key is typically used to iterate through the top-level items, and the up and down arrows are used to navigate sub-menus.
```
Comment (nitedog) [inline source/menus/application-menus.html.erb.md]: lol. Your fault! ;-)

File: source/menus/application-menus.html.erb.md (hunk 2 above)
Comment (yatil) [inline source/menus/application-menus.html.erb.md]: I’ve already proposed changes to this PR in that PR: https://github.com/w3c/wai-tutorials/pull/439

see: https://github.com/w3c/wai-tutorials/pull/439/files

I was under the impression I have let you know about those PR (maybe via mail?) but I don’t see it referenced here. Sorry for the confusion!

(Yes, that is a Pull Request for a Pull Request. Git to the max!)

**Comment by yatil** [comment]
PR for this item in #439.

**Pull request merged**